### ISSUE TITLE
Keyboard navigation in new history: history controls

### ISSUE THREAD

**Issue by elichad** [galaxyproject/galaxy#13937]
**Describe the bug**

*Edit: oops, I thought I was testing on dev but I was actually on 22.01. Looks like a bunch of these have been fixed, yay*

- When pressing the history edit button, a form appears to update the name and annotation, but the fields are before the edit button in the tab order, so you have to tab backwards to get to them - this is confusing.
- Similar issue where the filter field is not adjacent to the filter button in tab order
- Created tags have no focus indicator
- Created tags can't be deleted using keyboard
- Cannot tab to 'X' button in tag entry field
- Cannot tab to the panel collapse button or the resize control - these also don't seem to have accessible names or be labeled as buttons in any way

**Galaxy Version and/or server at which you observed the bug**
Galaxy Version: dev
Commit: 2f01b32b948a85e0ce5ef916aa7b66597ab94371

**To Reproduce**
1. Activate new history
2. Tab to history edit button & press
3. Tab backwards to find the form
4. Enter some tags
5. Tab to a created tag and try to delete it (enter, space, delete) - nothing happens
6. Try to tab to the 'X' in the tag field - cannot reach it

**Expected behavior**

- Upon pressing 'Edit,' the history name field appears next in the tab order.
- Created tags can be deleted using keyboard control
- Tag field can be cleared using keyboard control (actually, is this even what the 'X' is meant to do? I realised it behaves slightly different to my expectation)
- Can collapse & resize history panel using keyboard control
